spec/features/display_active_and_retired_products_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+feature "a user" do
+  let(:category) do
+    Category.create(name:        "Plants",
+                    description: "Plants category description",
+                    slug:        "plants")
+  end
+
+  before do
+    category.products.create(
+                 name:        "Plant 1",
+                 description: "This is the description for plant 1",
+                 price:       19.99,
+                 image_url:   "plants/plant-2.jpg",
+                 status:      "active")
+    category.products.create(
+                 name:        "Plant 2",
```

Comment:
Unnecessary spacing detected.<br>Indent the first parameter one step more than the previous line.